examples/http-certification/custom-assets/README.md

```diff
@@ -333,9 +333,11 @@ Lastly, a function specifically to certify the `index.html` file. Since the fron
 This will allow the canister to return this file for any path that does not exactly match an existing path in the tree. If the canister tries to return this file instead of an exact match that exists, verification will fail.
 
 ```rust
-const INDEX_REQ_PATH: &str = "";
-const INDEX_TREE_PATH: HttpCertificationPath = HttpCertificationPath::Wildcard(INDEX_REQ_PATH);
-const INDEX_FILE_PATH: &str = "index.html";
+lazy_static! {
```

Comment:
Since we're not directly using the enum anymore and we're using factories we can't use `const`